instructor/process_response.py

```diff
@@ -664,6 +664,9 @@ def handle_response_model(
     else:
         raise ValueError(f"Invalid patch mode: {mode}")
 
+    if "messages" in new_kwargs:
```

Comment:
Implemented a check to see if "messages" in new_kwargs

https://github.com/jxnl/instructor/actions/runs/11197184367/job/31126830225?pr=1045#step:7:164